From 8ace4fc9367b92759d7430b0da689c08fa0b6fef Mon Sep 17 00:00:00 2001
From: Pandith N <pandith.n@intel.com>
Date: Thu, 22 Apr 2021 18:35:23 +0530
Subject: [PATCH 064/109] SPI-DMA support for 32 bit transfers

Corrected APB register setting for hardware handshake interface.
DMA channel configuration, APB slot number needs to be programmed to
SRC/DST hardware handshake interface.
Verified SPI Tx and Rx with DMA for 32 bit mode.

Signed-off-by: Pandith N <pandith.n@intel.com>
---
 drivers/dma/dw-axi-dmac/dw-axi-dmac.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/dma/dw-axi-dmac/dw-axi-dmac.h b/drivers/dma/dw-axi-dmac/dw-axi-dmac.h
index 380005afde16..4bb2210cc23a 100644
--- a/drivers/dma/dw-axi-dmac/dw-axi-dmac.h
+++ b/drivers/dma/dw-axi-dmac/dw-axi-dmac.h
@@ -258,8 +258,8 @@ enum {
 
 /* CH_CFG_H */
 #define CH_CFG_H_PRIORITY_POS		17
-#define CH_CFG_H_DST_PER_POS		12
-#define CH_CFG_H_SRC_PER_POS		7
+#define CH_CFG_H_DST_PER_POS            12
+#define CH_CFG_H_SRC_PER_POS            7
 #define CH_CFG_H_HS_SEL_DST_POS		4
 #define CH_CFG_H_HS_SEL_SRC_POS		3
 enum {
-- 
2.25.1

